Sazabi.rc

```diff
@@ -1590,11 +1590,12 @@ END
 
 IDD_SETTINGS_DLG_LOG DIALOGEX 0, 0, 370, 291
 STYLE DS_SETFONT | DS_CONTROL | WS_CHILD | WS_CAPTION
-CAPTION "Logging"
+CAPTION "s"

```

Comment:
Can we revert this CAPTION's change?
I think "Logging" is better.

Reply:
This is unintended change, I'll fix it.

Reply:
Fixed.